### ISSUE TITLE
Without Consul how can i use Fabio?

### ISSUE THREAD

**Issue by Rachitmehrotra** [fabiolb/fabio#549]
Hello Team ,

How can i use fabio with consul i need to use it direct without using consul is there any way or if you guys provide any documentation for that ?

**Comment by tristanmorgan** [comment]
Fabio is tightly integrated with Consul is doesn't really make sense to run it without Consul. If you are using a different service-discovery tool there is probably an alternative load balancer better suited.
Disclosure: I'm not from the Fabio Team but I love the tool.

**Comment by magiconair** [comment]
@Rachitmehrotra you can configure static routes directly in `fabio.properties` or point to a file with routes:

https://fabiolb.net/ref/registry.backend/
https://fabiolb.net/ref/registry.file.path/
https://fabiolb.net/ref/registry.static.routes/

**Comment by MateusMac** [comment]
Hi Guys,
I'm trying to use a file with router, but no success so far.

My filename is config.routes. I have set the registry.file.path in the fabio.properties and this is the code I ran:

`fabio -cfg C:\Users\dev01\go\src\github.com\fabiolb\fabio\fabio.properties`

But I'm getting this error:

`[WARN] Error initializing backend. open ".\config.routes"`
`Error initializing backend. open ".\config.routes": A syntax for the file name, directory name, or volume label is incorrect.`

And here is what I put inside the file:

`route add service-a / http://host-a:11050/ tags "a,b"`
`route add service-b /test http://host-a:11051/ tags "c,d"`

Any Ideas?
